### ISSUE TITLE
Actual published app

### ISSUE THREAD

**Issue by ChrisMacNaughton** [cantino/post_location#2]
Any chance you could compile it and make that available for download?  I don't want to buy / install RubyMotion just to do this!

**Comment by benjamreynolds** [comment]
@cantino, hate to dig up an old thread but, is there a status on this?

**Comment by cantino** [comment]
No, sorry, I'm not supporting the app anymore. If someone wants to take the source code and push it, I'm totally open to that. I believe some people are using something called PostGPS.
